src/models/entity/summary.js

```diff
@@ -0,0 +1,21 @@
+// Entity summary
+const DATASOURCES = {
+  NCBIGENE: 'http://identifiers.org/ncbigene/',
+  HGNC: 'http://identifiers.org/hgnc/',
+  UNIPROT: 'http://identifiers.org/uniprot/',
+  GENECARDS: 'http://identifiers.org/genecards/'
```

Comment:
Should these be in conf?

Reply:
Yeah we need to put these somewhere global and integrate similar stuff including
  - `src/client/common/external-service-data.js` #1108  
  - `src/models/datasources.js` 

Will deal with as part of #1108.